model/transformers/README.md

```diff
@@ -7,7 +7,7 @@ Currently, dffml_model_transformers supports HuggingFace Transformers tensorflow
 
 ## Documentation
 
-Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers
+Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers 
```

Comment:
Adding a space at the end of this line causes a whitespace error. We need to remove the space at the end of line 6

```suggestion
Documentation is hosted at https://intel.github.io/dffml/plugins/dffml_model.html#dffml-model-transformers
```

Reply:
whoops, I didn't know that invisible whitespace can cause a problem. ;)
Thanks